account/key/storage.go

```diff
@@ -0,0 +1,15 @@
+package key
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+)
+
+type Identity = []byte
```

Comment:
Any reason for adding this new type instead of reusing `Address` (from key/address.go)?

Reply:
I've just replaced Address with Identity in 1ce93068f62e294417aa5ab2e3778ae47ecc39d0

Address and Identity are both just a byte array. But the Address has less meaning. I intend to cover more meaning (eg. alias).

Reply:
Okay, good!